_pages/en/it-picture-long-term.md

```diff
@@ -87,35 +87,59 @@ Ecosystem of services are created by leveraging private sector expertise where o
 
 #### Modern Open Organization
 
-IITB is a modern open organization with a very open culture.
+##### Enterprise Architecture
+
+Enterprise Architecture (EA) committees and review boards at IITB-ESDC have aligned with TBS mandatory procedures for EA (users, open standards, open source software and cloud first).
 Wherever possible code and projects are developed in the open and use of open source software is the norm.
-IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
-IITB is responsible for providing the automated services required to run a secure and agile business.
-This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment by which business may securely deploy their own solutions.
+EA teams enforce a loosely coupled architecture that empowers development and operations teams to meet business needs.
+
+Where possible, services are hosted on cloud infrastructure using containers. (TODO: find cloud adoption rates)
 
-- 90% of employees are able to work from anywhere (TODO: validate measure with trends)
-- Internal and external services are available through APIs
-- 98% of services are running on public cloud infrastructure (TODO: find cloud adoption rates)
-- Workloads that are classified Protected B and below are hosted on the public cloud.
-  - Classified workloads (secret information) on SSC data centres
-- IITB serves as a stewart of IT in ESDC and has embedded IT expertise in program areas (since it has moved to DevOps and Product Management).
-  - As a stewart, IITB provides ESDC Programs and their employees with the following services:
-    - Corporate services (hardware and software support)
-    - Cyber security
-    - Technology architecture
-    - Development teams
-- Automation of internal services
-- Leveraging IT services (from IITB) in a self-serving method
+Internal and external services are developed and supported as APIs first.
+APIs for services to Canadians are made available through the API store.
+All user interfaces (UI) developed use existing APIs and don't interact with the database directly.
```

Comment:
"and don't interact with the database directly." ... Goodness, I certainly hope so. Feel like we should be able to remove this?

Reply:
change 
All user interfaces (UI) developed use existing APIs and don't interact with the database directly.
to
User Interfaces (UI) are completely decoupled from the service interfaces. Allowing for different UIs to be built and interact with the service according to the channel  of choice of the end-user.

Reply:
I added an "all" :man_facepalming:

Now removed